doc/how_to/callbacks/load.md

```diff
@@ -1,41 +1,180 @@
+![panel-defer-all-example](https://user-images.githubusercontent.com/42288570/245752511-b2970c4c-7144-4b1a-af36-4c90b1873de6.gif)
+
+## Defer Specific Tasks
+
+Its also easy to defer the execution of selected, displayed tasks with `pn.panel(..., defer_load=True)`.
+
+```python
+import time
+import panel as pn
+
+pn.extension(loading_indicator=True, template="bootstrap")
+
+
+def short_running_task():
+    return "# I'm shown on load"
+
+
+def long_running_task():
+    time.sleep(3)
+    return "# I'm deferred and shown after load"
+
+
+pn.Column(
+    short_running_task,
+    pn.panel(long_running_task, defer_load=True, min_height=50, min_width=200),
+).servable()
+```
+
+![panel-defer-specific-example](https://user-images.githubusercontent.com/42288570/245752506-9ac676e9-65b2-4d9d-a01a-ce01a12dfda4.gif)
+
+## Defer Tasks That are not Displayed
+
+So far the tasks have been producing and returning viewable objects to be displayed in a your app. Sometimes that is not what you want. Sometimes you want to load or pre-compute one or more parameter values, send a message somewhere or similar.
+
+For those use cases you can use `pn.state.onload` to run one or more tasks when the application has loaded.
+
+```python
+import time
+import panel as pn
+import pandas as pd
+import param
+
+pn.extension(loading_indicator=True, template="bootstrap")
+
+class AppState(param.Parameterized):
+    data = param.DataFrame()
+
+def short_running_task():
+    return "# I'm shown on load"
+
+state = AppState()
+
+def update():
+    time.sleep(2)
+    state.data = pd.DataFrame({"x": [1, 2, 3, 4], "y": [1, 3, 2, 4]})
+
+pn.state.onload(update)
+
+pn.Column(
+    short_running_task,
+    pn.pane.DataFrame(state.param.data),
+).servable()
+```
+
+![panel-defer-onload-example](https://user-images.githubusercontent.com/42288570/245752503-a8042242-503e-4ba3-ad24-1f1a16aac058.gif)
+
+## Defer and Orchestrate Dependent Tasks
+
+Sometimes you have multiple tasks that depend on each other and you need to *orchestrate* them. To handle those scenarios you can combine what you have learned so far with `pn.bind`.
+
+Lets take an example where we
+
+- load a shared dataset.
+- display the dataset in a Table
+- transform the dataset and display it as a plot
+
+```python
+import time
+import panel as pn
+import pandas as pd
+import param
+import hvplot.pandas
+
+pn.extension(sizing_mode="stretch_width", template="bootstrap", theme="dark")
+
+class AppState(param.Parameterized):
+    data = param.DataFrame()
+
+    def update(self):
+        time.sleep(2)
+        state.data = pd.DataFrame({"x": [1, 2, 3, 4], "y": [1, 3, 2, 4]})
+
+def loading_indicator():
+    return pn.indicators.LoadingSpinner(value=True, height=25, width=25, align="center")
+
+def short_running_task():
+    return "# I'm shown on load"
+
+def table(data):
+    if data is None:
+        return pn.Row(loading_indicator(), "Loading data")
```

Comment:
Should be able to simply give the loading indicator a name now.

Reply:
At least when #5194 has been committed to `main`.